bin/main.dart

```diff
@@ -2,6 +2,6 @@ import 'package:flutter_launcher_icons/constants.dart';
 import 'package:flutter_launcher_icons/main.dart' as flutter_launcher_icons;
 
 void main(List<String> arguments) {
-  print(introMessage('0.8.0'));
+  print(introMessage);
```

Comment:
This and some additional tests added to the test suite should address #207...

Reply:
Thanks for the fix! 🎉